packages/create-cloudflare/src/workers.ts

```diff
@@ -147,8 +151,8 @@ async function updateFiles(ctx: Context) {
 	contents.wranglertoml = contents.wranglertoml
 		.replace(/^name = .+$/m, `name = "${ctx.project.name}"`)
 		.replace(
-			/^compatibility_date = .+$/m,
-			`compatibility_date = "${new Date().toISOString().substring(0, 10)}"`
+			/^compatibility_date\s=\s"<TBD>"$/m,
```

Comment:
Do you need to update the name's `.replace` too?

Reply:
Good spot. I'll add that.

Reply:
I also noticed that my regex was a bit too restrictive regarding the spaces around the `=`.